gitian-building/gitian-building-setup-gitian-fedora.md

```diff
@@ -38,9 +38,7 @@ echo 'exit 0' >> /etc/rc.d/rc.local
 # mark executable so that the rc.local-service unit gets pulled automatically
 # into multi-user.target
 chmod +x /etc/rc.d/rc.local
-# make sure that USE_LXC is always set when logging in as gitianuser,
-# and configure LXC IP addresses
-echo 'export USE_LXC=1' >> /home/gitianuser/.bash_profile
```

Comment:
Removing this here would mean it had to be set when manually building (not using the python script)?

Reply:
I think it would be easier for review to split the two changes into two pull requests